hexapod_dart/src/hexapod_dart_simu.cpp

```diff
@@ -334,3 +291,86 @@ void HexapodDARTSimu::_add_floor()
 
     _world->addSkeleton(floor);
 }
+
+double HexapodDARTSimu::_min_dist_angle(double a1, double a2)
```

Comment:
I took the idea from here: http://stackoverflow.com/questions/1878907/the-smallest-difference-between-2-angles . Also, this one suggests the same: http://stackoverflow.com/questions/7570808/how-do-i-calculate-the-difference-of-two-angle-measures . Made some quick tests and seems valid.

Another implementation could be: `atan2(sin(x-y), cos(x-y))`.